.github/workflows/deploy-hubs.yaml

```
@@ -69,41 +67,44 @@ jobs:
 
       - name: Check which cluster directory has changes (if any)
         uses: dorny/paths-filter@v2
-        id: config_files
+        id: cluster_specific_files
         with:
           filters: |
-            hub_config:
+            changes:
               - "config/clusters/${{ matrix.cluster_name }}/**"
 
+      # To continue this cluster specific job, there should have been
+      # changes to the cluster common files or cluster specific files.
+      - name: Decide if the job should continue
+        id: decision
+        run: |
+          echo ::set-output name=continue-job::${{ (steps.cluster_common_files.outputs.files == 'true' || steps.cluster_specific_files.outputs.changes == 'true') }}
```

Removing a redundant parentheses 

```suggestion
          echo ::set-output name=continue-job::${{ steps.cluster_common_files.outputs.files == 'true' || steps.cluster_specific_files.outputs.changes == 'true' }}
```